rfc-003-adopt-deep-work-as-team-norm.md

```diff
@@ -0,0 +1,31 @@
+# Adopt Deep Work as a Team Norm
+
+## Summary
+
+Adopt the discipline of [Deep Work](https://blog.evernote.com/blog/2017/02/23/deep-work-matters-distracted-world/) as a [team norm](https://github.com/madetech/handbook#team-norms).
+
+
+## Problem
+
+Our modern work environment can often make it hard for us to engage in deep work. Between our smartphones, Slack, email, distractions in the office, the doorbell etc. it can tricky to find sufficient space to get uninterrupted periods of focus.
+
+As an organisation we also explictly value and encourage customer interaction. We like to make ourselves available to have fluid conversations with our customers throughout the day.
+
+I hypothesise that this makes us less efficient as a business. We deliver less value to our customers than we're capable. In many cases it's holding back our personal development by not providing enough thinking time to get really absorbed in to solving hard problems.
+
+I believe our environment unconsciously encourages shallow work. I believe conscious action is needed to widely embrace longer periods of high focus.
+
+Further reading:
+
+- [Deep work at Trello](https://blog.trello.com/deep-work-productivity-interview-with-cal-newport?utm_source=newsletter&utm_medium=email&utm_campaign=december2017_ENnewsletter)
+- [Deep work at Evernote](https://blog.evernote.com/blog/2017/02/23/deep-work-matters-distracted-world/)
+
+## Proposal
+
+An element of shallow work is necessary for most roles within the business. Customers need to be conversed with, emails (sometimes) need to be answered, it's fun and rewarding to chat with peers on Slack.
+
+In their next 121, every member of the team collates a Deep Work plan. This involves the team member identifying the tasks that they typically need to perform in the week. The team member then suggests an optimal ratio between deep and shallow work for them to optimally perform their role.
+
+As part of the Delivery Healthcheck that teams will be reporting on from next year, deep work for all members of the team will tracked according to the individuals ideal ratio.
```

Comment:
- will *be* tracked
- apostrophe in individual's

Reply:
How are we going to be tracking deep work?

Reply:
@camillavk IDK. Requesting comments :-)